.github/workflows/interpreter.yml

```
@@ -0,0 +1,64 @@
+name: Interpreter Test
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SPEC_SPLIT_DOTS: 160
+
+jobs:
+  test-interpreter_spec:
+    runs-on: ubuntu-22.04
+    container:
+      image: crystallang/crystal:1.7.3-build
+    name: "Test Interpreter"
```

I'd avoid redundant naming (workflow name points at the Interpreter already)

```suggestion
    name: Test
```